utils.go

```diff
@@ -116,13 +116,28 @@ func (lo *LatencyOutput) Show(regions *AWSRegions) {
 	}
 }
 
-// GetRegions returns a list of regions
+// GetRegions returns a list of all regions
 func GetRegions() AWSRegions {
+	us := GetRegionsUS()
+	other := GetNonUSRegions()
+
+	return append(us, other...)
+}
+
+// GetRegionsUS returns a list of regions
```

Comment:
Function is self explanatory, so comment isn't necessary. 
What's the best way to edit/append a PR?